### DISCUSSION TITLE
Vetur can't find package.json in /xxxx/xxxxxx.

### DISCUSSION THREAD

**Discussion by cottrell** [vuejs/vetur#2563] [Q&A] [answered]
There is a section in the docs that does not actually tell you anything about how to point vetur at the package.json. 

I open vscode in a top level and have many projects of different kinds below. I suspect vetur doesn't know how to look to find "the current" project? Not sure. I tried to open a new window on that project but it still failed.

**Comment by yoyo930021** [comment, marked as answer]
[Ref1](https://vuejs.github.io/vetur/reference/#detail) [Ref2](https://vuejs.github.io/vetur/guide/FAQ.html#vetur-can-t-find-package-json-in-xxxx-xxxxxx)

I'm sorry to have caused you distress.
This issue is improve docs about this problem.
https://github.com/vuejs/vetur/issues/2543

I need some help about this.
https://github.com/vuejs/vetur/issues/2537#issuecomment-742212004

**Comment by mariusa** [comment]
Hi, I came here for the same issue.

With vscode I Open Folder `/work` containing multiple projects:
`work/admin`, `/work/project2` ...

Some of these projects are Vue and have package.json 

How could Vetur handle this instead of complaing that `/work` doesn't have package.json  ?

Thanks @yoyo930021

**Reply by ip75** [reply]
I have the same issue

**Reply by gertvdijk** [reply]
Oh wow, thanks for this hint, I did not realize the `package.json` is only found _at the root_ of the opened folder. I've been banging my head against the wall for this. My project has the frontend part located in a subdir and the `package.json` file IS JUST THERE and it was really _only Vetur_ complaining. The docs could definitely be improved to include this limitation.
